### ISSUE TITLE
Rendering the spec with internalized external references

### ISSUE THREAD

**Issue by yozel** [pb33f/libopenapi#222]
I'm trying to migrate https://github.com/deepmap/oapi-codegen into libopenapi and I just need to implement one last functionality. Is it possible to render a document as a yaml/json with internalized external references so it would not have any external dependencies? `kin-api` has  `swagger.InternalizeRefs` function for that ([example usage](https://github.com/yozel/oapi-codegen/blob/7bb8bdc2adb3b7180b2aeb2b09131af02f0c5475/pkg/codegen/inline.go#L31)). I was wondering if there's such a thing exist in `libopenapi`?

Example:
From
```json
{
    "openapi": "3.0.0",
    "info": {},
    "paths": {},
    "components": {
        "schemas": {
            "Container": {
                "properties": {
                    "object_a": {
                        "$ref": "./packageA/spec.yaml#/components/schemas/ObjectA"
                    },
                    "object_b": {
                        "$ref": "./packageB/spec.yaml#/components/schemas/ObjectB"
                    },
                    "object_c": {
                        "$ref": "./object_c.json"
                    }
                }
            }
        }
    }
}
```

to 

```json
{
    "components": {
        "schemas": {
            "Container": {
                "properties": {
                    "object_a": {
                        "$ref": "#/components/schemas/ObjectA"
                    },
                    "object_b": {
                        "$ref": "#/components/schemas/ObjectB"
                    },
                    "object_c": {
                        "$ref": "#/components/schemas/object_c"
                    }
                }
            },
            "ObjectA": {
                "properties": {
                    "name": {
                        "type": "string"
                    },
                    "object_b": {
                        "$ref": "#/components/schemas/ObjectB"
                    }
                }
            },
            "ObjectB": {
                "properties": {
                    "name": {
                        "type": "string"
                    }
                }
            },
            "object_c": {
                "additionalProperties": true,
                "type": "object"
            }
        }
    },
    "info": {
        "title": "",
        "version": ""
    },
    "openapi": "3.0.0",
    "paths": {}
}
```

**Comment by yozel** [comment]
I think that would be the same request, yes.

As a proof-of-concept, I created this: https://github.com/yozel/libopenapi-internalize

I used `index.Rolodex` and `index.Index`, but not low-level or high-level models since I couldn't find a way to properly use them. Let me know if you have suggestions so I can work on the idea/feature.

**Comment by daveshanley** [comment]
This feature is coming in the next release of libopenapi.

**Comment by daveshanley** [comment]
This feature is now available in `v0.15.0`

https://pb33f.io/libopenapi/bundling/
